dev/com.ibm.ws.microprofile.rest.client.cdi/src/com/ibm/ws/microprofile/rest/client/cdi/LibertyRestClientExtension.java

```diff
@@ -91,5 +96,24 @@ public void registerClientBeans(@Observes AfterBeanDiscovery afterBeanDiscovery,
 
     public void registerErrors(@Observes AfterDeploymentValidation afterDeploymentValidation) {
         errors.forEach(afterDeploymentValidation::addDeploymentProblem);
+        if (!requestScopedInterfaces.isEmpty()) {
+            
+            Tr.info(tc, "rest.client.interface.using.request.scope", 
+                    requestScopedInterfaces.stream().map(c -> c.getName()).collect(Collectors.joining(" ")));
+        }
+        
+    }
+
+    public void logRequestScopedClientInterfaces(@Observes ProcessBean<?> processBean) {
```

Comment:
Any need for null checks for processBean or bean?

Reply:
According to the CDI docs/examples, the processBean should always be non-null.